src/main/java/hudson/util/VersionNumber.java

```diff
@@ -513,6 +513,9 @@ public boolean isNewerThan(VersionNumber rhs) {
         return compareTo(rhs) > 0;
     }
 
+    /**
+     * @deprecated see {@link #getDigitAt(int)}
```

Comment:
@jglick I tried to figure out what `digit()` did without success, returned always the first digit and its input validation was poor.